### PR TITLE
Cost model diff

### PR DESCRIPTION
**Pull request by jamesgorrie**
Adding the new flag for diffing against the new cost model.
I added some more config / setup stuff too to get it working across the board.

### CODE REVIEW

File: components/checkbox/flag.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const $combine = Rx.Observable.combineLatest;
+
+export function flagCheckbox(label, initialState = false) {
+    const cb = labelledCheckbox(label, initialState);
+    return {
+        model: {
+            value$: cb.model.checked$.map(checked => checked ? true : undefined)
+        },
+        tree$: $combine(cb.tree$, (...views) => h('span', views))
```
Comment (jamesgorrie) [inline components/checkbox/flag.js]: Sure this doesn't need to be a `$combine`... perhaps `cb.tree$.map(view => h(span, [view]))`?

File: components/checkbox/flag.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import h from 'virtual-dom/h';
+
+import Rx from 'rx';
+
+import {labelledCheckbox} from './labelled';
+
+
+const $combine = Rx.Observable.combineLatest;
+
+export function flagCheckbox(label, initialState = false) {
```
Comment (theefer) [inline components/checkbox/flag.js]: Fun, but why not use a simple `labelledCheckbox`?

File: config.js [hunk 3]
```diff
@@ -1,22 +1,20 @@
 System.config({
-  "baseURL": "/rights-inspector",
-  "transpiler": "traceur",
-  "paths": {
-    "*": "*.js",
-    "github:*": "jspm_packages/github/*.js",
-    "npm:*": "jspm_packages/npm/*.js"
-  }
-});
+  //baseURL: '/rights-inspector',
```
Comment (theefer) [inline config.js]: We need to leave that in for when we deploy. I know it's not compatible with dev locally but I'd rather keep the code broken for dev than for prod. Obv we should find a way to fix this nicely.

Comment (theefer) [inline config.js]: Oh just saw the line below..

**Comment by theefer** [comment]
Looks good, just one comment about the added component.

File: components/checkbox/flag.js (hunk 2 above)
Comment (jamesgorrie) [inline components/checkbox/flag.js]: It was when this was going to be a flag - it's still handy as it constructs the URL:

```
checked    => ?free=true&costModelDiff=true
unchecked  => ?free=true
```

Comment (jamesgorrie) [inline components/checkbox/flag.js]: Seems nicer to do here than in the [`media-api` URL construction](https://github.com/guardian/grid-inspector/blob/master/media-api.js#L35-L48).

File: config.js (hunk 3 above)
Comment (jamesgorrie) [inline config.js]: Yea - not great, but better than commenting, committing, remembering to uncomment, uncomment, commit.

File: components/checkbox/flag.js (hunk 2 above)
Comment (theefer) [inline components/checkbox/flag.js]: Is it still needed now it's a proper boolean flag?

File: config.js (hunk 3 above)
Comment (theefer) [inline config.js]: :+1:

File: components/checkbox/flag.js (hunk 2 above)
Comment (jamesgorrie) [inline components/checkbox/flag.js]: Nope, as it would explicitly set it to false... I'll remove.

**Comment by jamesgorrie** [comment]
Removed `flag`...

File: package.json [hunk 4]
```diff
@@ -29,8 +29,11 @@
...
     },
     "devDependencies": {
-      "traceur": "github:jmcriffey/bower-traceur@0.0.87",
-      "traceur-runtime": "github:jmcriffey/bower-traceur-runtime@0.0.87"
+      "traceur": "github:jmcriffey/bower-traceur@0.0.91",
+      "traceur-runtime": "github:jmcriffey/bower-traceur-runtime@0.0.91"
     }
+  },
+  "scripts": {
+    "setup": "jspm install"
```
Comment (theefer) [inline package.json]: or `postinstall` as we do elsewhere?

**Comment by theefer** [comment]
Minor comment on script which could be even further simplified, else :+1:

File: README.md [hunk 5]
```diff
@@ -4,3 +4,13 @@ Web UI to inspect metadata and rights of Grid images.
...
 This project uses an experimental combination of RxJS and virtual-dom
 as base for its application architecture.
+
+# Setup
+
+Get your local `/config/picdar.json` and `/config/media-api.json` from
+a friendly developer near you.
+
+    $ npm install
+    $ npm run setup
```
Comment (theefer) [inline README.md:14]: No longer needed btw?

**Pull request merged**